compose/broker.xml

```diff
@@ -0,0 +1,145 @@
+<?xml version='1.0'?>
```

Comment:
I got this base of this file by looking at the auto generated one in a blank service and then tweaking it.

I left lots of the comments in it, but also removed unnecessary details. Values not specified should fallback on their default